jep/0000/README.adoc

```diff
@@ -0,0 +1,366 @@
+
+=== Scope of changes
+
+* Jenkins core
+* Jenkins remoting
+* Jenkins configuration as code
+* Docker packaging
+* Plugins
+* Jenkins website: www.jenkins.io
+* Jenkins CI instances:
+    link:https://ci.jenkins.io/[ci.jenkins.io],
+    infra.ci.jenkins.io,
+    trusted.ci.jenkins.io,
+    link:https://www.jenkins.io/security/#team[Security team]'s instance
+* Maven build flow (
+    link:https://github.com/jenkinsci/maven-hpi-plugin[Maven HPI Plugin],
+    link:https://github.com/jenkinsci/plugin-pom[Plugin POM],
+    link:https://github.com/jenkinsci/pom[Jenkins POM],
+    link:https://github.com/jenkinsci/bom[Jenkins plugin bill of materials POM],
+    etc.)
+* Test tools:
+    link:https://github.com/jenkinsci/jenkins-test-harness[Jenkins Test Harness (JTH)],
+    link:https://github.com/jenkinsci/acceptance-test-harness[Acceptance Test Harness (ATH)],
+    link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)]
+* User and Developer Documentation
+
+Out of the scope for this JEP:
+
+* Packaging in subprojects (unless specifically noted): Jenkins Operator, Jenkins Evergreen, Jenkinsfile Runner, etc.
+  They will be handled in follow-up JEPs if needed.
+* Gradle build flow
+
+=== Jenkins core patches
+
+Work to be considered is defined in link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688].
+
+==== Library updates
+
+* The link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688 epic] will include library updates as needed.
+* Some updates may require downstream plugin updates.
+
+==== Core patches
+
+* link:https://github.com/jenkinsci/jenkins/pull/6083[PR 6083] - Drop Java 8 from Jenkins core
+* link:https://github.com/jenkinsci/jenkins/pull/6086[PR 6086] - Switch Jenkins acceptance test harness in core to Java 11
+* link:https://github.com/jenkinsci/jenkins/pull/6092[PR 6092] - Add administrative monitor to remind of upcoming end of Java 8 support
+
+==== Build flow updates
+
+* Jenkinsfile is updated to stop running tests with Java 8
+** It includes Unit tests, JTH and ATH smoke tests
+* It is possible to build Jenkins Core with the release profile on Java 8
+
+=== Jenkins Docker packaging
+
+The containers tagged for Java 8, like `latest-jdk8` and `centos7-jdk8` will no longer be updated.
+The upgrade guide and the announcement blogpost will note that users must switch to other images.
+Labels will not be removed for existing containers, but those labels will not be provided for new builds.
+
+Java 8 images will no longer be provided for the controller containers:
+
+* https://hub.docker.com/r/jenkins/jenkins[Controller]
+
+==== Jenkins Agent Images
+
+Java 8 images will no longer be provided for the general purpose agent containers:
+
+* https://hub.docker.com/r/jenkins/agent[Agent]
+* https://hub.docker.com/r/jenkins/inbound-agent[Inbound agent]
+* https://hub.docker.com/r/jenkins/ssh-agent[Outbound (ssh) agent]
+
+Tool specific agent containers will no longer include Java 8:
```

Comment:
Agreed.  They seem to be updated regularly with the latest base image, but only have tags for `latest`.

I didn't put text into this about the state of those images because I thought it was a separate topic.